code/game/machinery/engi_points_vendor.dm

```diff
@@ -0,0 +1,197 @@
+		new /datum/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,					400,	1),
+		new /datum/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,								500,	1),
+		new /datum/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,								500,	1),
+		new /datum/GBP_equipment("50 cardboard sheets",			/obj/item/stack/sheet/cardboard/fifty,						500,	1),
+		new /datum/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,									600,	1),
+		new /datum/GBP_equipment("Hardsuit x3",			/obj/item/clothing/suit/space/hardsuit,								750,	3),
+		new /datum/GBP_equipment("Jetpack Upgrade x3",		/obj/item/weapon/tank/jetpack/suit,								1000,	3),
+		new /datum/GBP_equipment("Powertools x4",			/obj/item/weapon/storage/belt/utility/chief/full,				2000,	4),
+		new /datum/GBP_equipment("Freon Canister",			/obj/machinery/portable_atmospherics/canister/freon,			2500,	1),
+		new /datum/GBP_equipment("Prototype Canister",			/obj/machinery/portable_atmospherics/canister/proto,		2500,	1),
+		new /datum/GBP_equipment("Advanced Magboot x3",			/obj/item/clothing/shoes/magboots/advance,					3000,	3),
+		new /datum/GBP_equipment("Reflector Box x3",			/obj/structure/reflector/box,								3500,	3),
+		new /datum/GBP_equipment("Radiation Collector x3",			/obj/machinery/power/rad_collector,						4000,	3),
+		new /datum/GBP_equipment("ERT Hardsuit x5",		/obj/item/clothing/suit/space/hardsuit/ert/engi,					7000,	5),
+		new /datum/GBP_equipment("Portal Gun x5",			/obj/item/weapon/gun/energy/wormhole_projector,					8000,	5),
+		new /datum/GBP_equipment("Ranged RCD x4",			/obj/item/weapon/rcd/arcd,										9000,	4),
+		new /datum/GBP_equipment("Prototype Atmos Vehicle x2",			/obj/vehicle/space/speedbike/atmos,					10000,	2),
+		new /datum/GBP_equipment("Reactive Decoy Armor x5",		/obj/item/clothing/suit/armor/reactive/stealth,				11000,	5),
+		new /datum/GBP_equipment("Prototype Repair Vehicle x3",		/obj/vehicle/space/speedbike/repair,					15000,	3),
+		new /datum/GBP_equipment("Chrono Suit x5",			/obj/item/clothing/suit/space/chronos,							20000,	5),
+		new /datum/GBP_equipment("WHAT HAVE YOU DONE... x5",		/obj/vehicle/space/speedbike/memewagon,					30000,	5),
+		)
+	..()
+
+/obj/machinery/engi_points_manager/Destroy()
+	engi_points_list -= src
+	if(radio)
+		qdel(radio)
+		radio = null
+	return ..()
+
+
+/obj/machinery/engi_points_manager/power_change()
+	..()
+	update_icon()
+
+/obj/machinery/engi_points_manager/update_icon()
+	if(powered())
+		icon_state = initial(icon_state)
+	else
+		icon_state = "[initial(icon_state)]-off"
+
+/obj/machinery/engi_points_manager/attack_hand(mob/user)
+	if(..())
+		return
+	if(!allowed(user))
+		user << "<span class='warning'>The shopping interface logs out with a message: Insufficient Access.</span>"
+		return
+	interact(user)
+
+/obj/machinery/engi_points_manager/interact(mob/user)
+	var/dat

```

Comment:
for the third time:
1. all of the lines adding text remain the same (this means no adding new line breaks, leave them as they are right now)
2. you make this ```var/list/dat = list()```
3. for ```set_content(dat)``` have it instead be ```set_content(dat.Join())```